hack/generator/pkg/astmodel/armconversion/from_arm_conversion_function.go

```diff
@@ -0,0 +1,431 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package armconversion
+
+import (
+	"fmt"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astbuilder"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astmodel"
+	"go/ast"
+	"go/token"
+)
+
+func (c *ArmConversionFunction) asConvertFromArmFunc(
```

Comment:
All these methods are massive and take quite a bit of reading to figure out what they're doing. Can you refactor a lot of it into some helper functions?

Reply:
I can certainly try